frontend/website-redesign/src/Theme.re

```diff
@@ -18,41 +19,292 @@ module Typeface = {
 module MediaQuery = {
   let tablet = "(min-width:48rem)";
   let desktop = "(min-width:90rem)";
-  
+
   /** to add a style to tablet and desktop, but not mobile */
   let notMobile = "(min-width:23.5rem)";
-  
+
   /** to add a style just to mobile  */
-  let mobile= "(max-width:48rem)";
+  let mobile = "(max-width:48rem)";
 };
 
-// module Type = {
-//   let h1jumbo = ...;
-//   let h1 = ...;
-//   let h2 = ...;
-//   let h3 = ...;
-//   let h4 = ...;
-//   let h5 = ...;
-//   let h6 = ...;
-//   let pageLabel = ...;
-//   let label = ...;
-//   let buttonLabel = ...;
-//   let navLink = ...;
-//   let sidebarLink = ...;
-//   let tooltip = ...;
-//   let creditName = ...;
-//   let metadata = ...;
-//   let announcement = ...;
-//   let pageSubhead = ...;
-//   let sectionSubhead = ...;
-//   let paragraph = ...;
-//   let paragraphSmall = ...;
-//   let paragraphMono = ...;
-//   let quote = ...;
-// };
+/** this function is needed to include the font files with the font styles */
+let generateStyles = rules => (Css.style(rules), rules);
+
+module Type = {
+  let h1jumbo =
+    style([
+      Typeface.monumentGrotesk,
```

Comment:
If all the styles have `monumentGrotesk` as their font, would it make sense to create a global font style and include it in each type via `merge()`? This way, there is one central place that's defined for the font.